### ISSUE TITLE
[error] Could not connect to Selenium Server. Have you started the Selenium Server yet?

### ISSUE THREAD

**Issue by jibet** [SeleniumHQ/selenium#3749]
## Meta -
OS:  Windows 10 64bit

Selenium Version:  
IDE: 2.9.1
Standalone server: 3.3.1 

Browser:  Firefox 52.0.2 64bit

## Expected Behavior -
Run Selenium IDE testscript on Chrome
## Actual Behavior -
[error] Could not connect to Selenium Server. Have you started the Selenium Server yet?
## Steps to reproduce -
1. created test script via firefox selenium IDE.
2. Enable webdriver in selenium IDE, change to chrome
3. Run in cmd, **java -Dbwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone- 3.3.1.jar -port 5555**
Received following response:
===================================
E:\Data\Desktop\Selenium Webdriver\Installer>java -Dbwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone-3.3.1.jar -port 5555
18:58:31.375 INFO - Selenium build info: version: '3.3.1', revision: '5234b32'
18:58:31.376 INFO - Launching a standalone Selenium Server
2017-04-03 18:58:31.394:INFO::main: Logging initialized @227ms to org.seleniumhq.jetty9.util.log.StdErrLog
18:58:31.437 INFO - Driver class not found: com.opera.core.systems.OperaDriver
18:58:31.437 INFO - Driver provider com.opera.core.systems.OperaDriver registration is skipped:
Unable to create new instances on this machine.
18:58:31.438 INFO - Driver class not found: com.opera.core.systems.OperaDriver
18:58:31.438 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
18:58:31.444 INFO - Driver provider org.openqa.selenium.safari.SafariDriver registration is skipped:
 registration capabilities Capabilities [{browserName=safari, version=, platform=MAC}] does not match the current platform WIN10
2017-04-03 18:58:31.484:INFO:osjs.Server:main: jetty-9.2.20.v20161216
2017-04-03 18:58:31.513:INFO:osjsh.ContextHandler:main: Started o.s.j.s.ServletContextHandler@685cb137{/,null,AVAILABLE}
2017-04-03 18:58:31.599:INFO:osjs.AbstractConnector:main: Started ServerConnector@49993335{HTTP/1.1,[http/1.1]}{0.0.0.0:5555}
2017-04-03 18:58:31.600:INFO:osjs.Server:main: Started @433ms
**18:58:31.600 INFO - Selenium Server is up and running**
===============================================

4. run the testscript (tested working fine in firefox), received [error] Could not connect to Selenium Server. Have you started the Selenium Server yet?

5. this is how my testscript looks like
![image](https://cloud.githubusercontent.com/assets/26647918/24606765/512338ce-18a1-11e7-9826-c17e010332ae.png)

Could anyone help with this please?I am doing a self learning on selenium, but encountered some issues with it. I have tried to google this error message， the solution provided is either disabled webdriver or installed selenium server. 

Note, I have tested the selenium standalone server 2.53.1/3.1.0/3.2.0,  initially it keep returning error when try to run the selenium server in cmd : -
_a. port 4444 in used
b.Exception in thread "main" com.beust.jcommander.ParameterException: Was passed main parameter 'chromedriver.exe' but no main parameter was defined_

**Comment by jibet** [comment]
Can anyone help on this please?or please let me know if more informations needed. Thanks

**Comment by diemol** [comment]
> Run in cmd, java -Dbwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone- 3.3.1.jar -port 5555
I think the problem is that you are starting Selenium standalone with port 5555, and Selenium IDE is expecting it to be on port 4444.

**Comment by jibet** [comment]
Thanks for the reply. Initially I was trying to connect to port 4444, but
it keep throw me with error that this port is in used. I have googled a
solution for it, and I found one that suggested to change to a different
port. I basically have no idea how much it would affect by using different
port :S

Regards,
Kam Hoon

**Comment by jibet** [comment]
Hi @diemol could you advise, is that crucial that selenium server MUST connect via port 4444?many thanks.

**Comment by diemol** [comment]
Hi @jibet,

I haven't used Selenium IDE before, but I downloaded it and tried it for this one, it is quite nice.

So, to make it short, it worked for me. I think the problem you had is related to two things:

* Typo in the `-Dwebdriver.chrome.driver` flag.
```
java -Dbwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone- 3.3.1.jar -port 5555
``` 
Notice that you have the java option with a **-Db**, the **b** is a typo. 
* Starting the standalone server with port -5555. When I did that, I got the _Could not connect to Selenium Server. Have you started the Selenium Server yet?_. So I think it expects it to be on port 4444. 
A command that should work for you is:
```
java -Dwebdriver.chrome.driver=chromedriver.exe -jar selenium-server-standalone- 3.3.1.jar
``` 
Just make sure chromedriver is in the correct path.
